src/main/java/com/denizenscript/depenizen/bukkit/commands/worldedit/WorldEditCommand.java

```diff
@@ -0,0 +1,370 @@
+                + (file != null ? file.debug() : ""));
+
+
+
+        if (action.asString().equalsIgnoreCase("schematicpaste")) {
+            if (file == null) {
+                Debug.echoError(scriptEntry.getResidingQueue(), "File path not specified!");
+                return;
+            }
+            if (position == null) {
+                Debug.echoError(scriptEntry.getResidingQueue(), "Location not specified!");
+                return;
+            }
+            String directory = URLDecoder.decode(System.getProperty("user.dir"));
+            File file_load = new File(directory + "/plugins/Denizen/schematics/" + file + ".schem");
+            ClipboardFormat format = ClipboardFormats.findByFile(file_load);
+            if (format == null) {
+                Debug.echoError(scriptEntry.getResidingQueue(), "File not found!");
+                return;
+            }
+            Clipboard clipboard;
+            try (ClipboardReader reader = format.getReader(new FileInputStream(file_load))) {
+                clipboard = reader.read();
+            } catch (Exception e) {
+                Debug.echoError(scriptEntry.getResidingQueue(), "File did not load correctly or not found!");
+                return;
+            }
+            if (clipboard == null) {
+                Debug.echoError(scriptEntry.getResidingQueue(), "Clipboard not found!");
+                return;
+            }
+            ClipboardHolder holder = new ClipboardHolder(clipboard);
+            if (rotate != null) {
+                AffineTransform transform = new AffineTransform();
+                transform = transform.rotateY(rotate.asInt());
+                holder.setTransform(holder.getTransform().combine(transform));
+            }
+            World w = new BukkitWorld(position.getWorld());
+            boolean pastenoair = true;
+            if (noAir == null) {
+                pastenoair = false;
+            }
+            if (undoable != null) {
+                if (target == null) {
+                    Debug.echoError(scriptEntry.getResidingQueue(), "Target not found!");
+                    return;
+                }
+                com.sk89q.worldedit.entity.Player p = BukkitAdapter.adapt(target.getOfflinePlayer().getPlayer());
+                try (EditSession editSession = WorldEdit.getInstance().getEditSessionFactory().getEditSession(w, -1,null,p)) {
+                    Operation operation = holder
+                            .createPaste(editSession)
+                            .to(BlockVector3.at(position.getBlockX(), position.getBlockY(), position.getBlockZ()))
+                            .ignoreAirBlocks(pastenoair)
+                            .build();
+                    Operations.complete(operation);
+                    editSession.flushSession();
+                    WorldEdit.getInstance().getSessionManager().get(p).remember(editSession);
+                } catch (WorldEditException e) {
```

Comment:
Please conform to basic Denizen formatting standards

Reply:
I'm not sure what you mean here. Is it the inside of the `try` clause or do I need to change `WorldEditException` to `Exception` ?

Reply:
Basic formatting of a try/catch